.github/workflows/lint.yml

```diff
@@ -0,0 +1,33 @@
+name: Lint python projects
```

Comment:
do you want to add this to the list of required checks for PRs?

Reply:
how do we do that?

Reply:
In the settings  -> Branch protection rules -> Require status checks to pass before merging